test/functionalTest/cases/3190_notify_only_attributes_that_change/notify_only_attributes_that_change_erros.test

```diff
@@ -0,0 +1,187 @@
+# Copyright 2019 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Notify only attributes that change for NGSIv2 for error POST /v2/subscriptions, POST /v2/subscriptions, POST /v2/subscriptions.
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+accumulatorStart --pretty-print localhost $LISTENER_PORT
+
+--SHELL--
+
+#
+# 01. POST /v2/subscriptions, create subscription with attrs, exceptAttrs and onlyChangedAttrs.
+# 02. POST /v2/subscriptions, create subscription with onlyChangedAttrs.
+# 03. POST /v2/subscriptions, create subscription with attrs and onlyChangedAttrs set to incorrect value.
+#
+
+echo '01. POST /v2/subscriptions, create subscription with attrs, exceptAttrs and onlyChangedAttrs.'
+echo '============================================================================================='
+payload='{
+  "description": "A subscription to get info about Room1",
+  "subject": {
+    "entities": [
+      {
+        "id": "Room1",
+        "type": "Room"
+      }
+    ],
+    "condition": {
+      "attrs": [
+        "A","B","C"
+      ]
+    }
+  },
+  "notification": {
+    "http": {
+      "url": "http://localhost:'$LISTENER_PORT'/notify"
+    },
+	"attrs" : [
+	  "A","B"
+	],
+	"exceptAttrs" : [
+	  "C"
+	],
+    "onlyChangedAttrs":true
+  },
+  "expires": "2040-01-01T14:00:00.00Z",
+  "throttling": 5
+}'
+orionCurl --url /v2/subscriptions --payload "$payload"
```

Comment:
This step is not needed. Note that according to http://telefonicaid.github.io/fiware-orion/api/v2/stable/

> `attrs` or `exceptAttrs` (both cannot be used at the same time)

(Although I'm not sure right now if we have some .test covering that...)

Reply:
fixed in f9887f1